mapiproxy/servers/default/emsmdb/emsmdbp_object.c

```diff
@@ -3154,18 +3265,16 @@ _PUBLIC_ struct emsmdbp_object *emsmdbp_object_ftcontext_init(TALLOC_CTX *mem_ct
 static enum MAPISTATUS emsmdbp_object_sharing_metadata_property(struct emsmdbp_context *emsmdbp_ctx, struct emsmdbp_object *sharing_object, struct Binary_r **attach_bin, TALLOC_CTX *mem_ctx)
 {
 	struct Binary_r		       *bin;
-	char			       *xml;
-	struct emsmdbp_prop_index      prop_index;
+	char			       *xml, *recipients_attr, *providers_node;
```

Comment:
there are a lot of variable declarations sharing same line, not always but mostly when there is a relationship between the variables.